### ISSUE TITLE
[Pingvin-share] Upgrade docker image to 0.21.3 (or newer)

### ISSUE THREAD

**Issue by mossroy** [DLAKE-IO/helm-charts#15]
The Helm chart currently deploys v0.17.0 by default. Latest version is v0.21.3

The values.yaml allows to manually override the tag, but it's not enough: if you only do so, it complains it cannot write inside some subdirectories of /var/lib/nginx, and in /run/nginx

I managed to manually workaround that by adding some emptyDir volumes, but it's clearly not ideal:
```
extraVolumes:
  - name: var-lib-nginx
    emptyDir: {}
  - name: var-lib-nginx-logs
    emptyDir: {}
  - name: var-lib-nginx-tmp
    emptyDir: {}
  - name: run-nginx
    emptyDir: {}
extraVolumeMounts:
  - mountPath: "/var/lib/nginx"
    name: var-lib-nginx
  - mountPath: "/var/lib/nginx/logs"
    name: var-lib-nginx-logs
  - mountPath: "/var/lib/nginx/tmp"
    name: var-lib-nginx-tmp
  - mountPath: "/run/nginx"
    name: run-nginx
```

**Comment by mossroy** [comment]
Note that, with this version 0.21.2, the readinessProbe on / also generates this error in pod logs (every 10 seconds):
```
[Nest] 41  - 12/31/2023, 11:44:46 AM   ERROR [ExceptionsHandler] Cannot read properties of undefined (reading 'password')
TypeError: Cannot read properties of undefined (reading 'password')
    at UserController.getCurrentUser (/opt/app/backend/dist/src/user/user.controller.js:32:38)
    at /opt/app/backend/node_modules/@nestjs/core/router/router-execution-context.js:38:29
    at process.processTicksAndRejections (node:internal/process/task_queues:95:5)
```
It's possible to workaround that by using `/api/health` instead of `/` as the path of this readinessProbe

I do not have this error log with default 0.17.0

**Comment by drustan** [comment]
Hello! Thank you for your interest. I plan to update the chart later today to address this and your other reported issues. Cheers!

**Comment by mossroy** [comment]
This is fixed with v0.0.2 of your helm chart: many thanks!
